WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -665,9 +666,17 @@ public void onStart() {
 
     @Override
     public void setTitle(final String title) {
+        if (isAdded()) {
+            mToolbarTitle = title.isEmpty() ? getString(R.string.wordpress_as_site_title) : title;
```

Comment:
Since the original PR that introduced this string resource [had to be reverted](https://github.com/wordpress-mobile/WordPress-Android/issues/7874#issuecomment-401191164), let's make sure this new PR includes a new definition of the string resource, thanks!